app/models/employer_profile.rb

```diff
@@ -850,6 +850,46 @@ def trigger_notices(event)
     ShopNoticesNotifierJob.perform_later(self.id.to_s, event)
   end
 
+  def upload_document(file_path,file_name,subject,size)
+    #doc_uri = Aws::S3Storage.save(file_path,'id-verification')
+    #file = File.open(file_path, "r:ISO-8859-1")
+
+    tmp_file = "#{Rails.root}/tmp/#{file_name}"
+    id = 0
```

Comment:
What's your plan in this method? 

https://github.com/health-connector/enroll/pull/100/files

Harsha is doing it differently there.

And we're doing it completely differently in families controller

https://github.com/health-connector/enroll/blob/master/app/controllers/insured/families_controller.rb#L189

https://github.com/health-connector/enroll/blob/master/app/models/aws/s3_storage.rb

Isn't using the above service what was discussed?